network: set the 'Connection' HTTP header field to 'Close'

From: Christian Prochaska <christian.prochaska@genode-labs.com>

With the 'Keep-Alive' setting, loading the genode.org website with Arora
took a long time. There appeared to be no indication that all needed data
had been received.

From: Christian Prochaska <christian.prochaska@genode-labs.com>
---
 src/network/access/qhttpnetworkconnection.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/network/access/qhttpnetworkconnection.cpp b/src/network/access/qhttpnetworkconnection.cpp
index 350c307..419e456 100644
--- a/src/network/access/qhttpnetworkconnection.cpp
+++ b/src/network/access/qhttpnetworkconnection.cpp
@@ -217,7 +217,7 @@ void QHttpNetworkConnectionPrivate::prepareRequest(HttpMessagePair &messagePair)
 #endif
         value = request.headerField("connection");
         if (value.isEmpty())
-            request.setHeaderField("Connection", "Keep-Alive");
+            request.setHeaderField("Connection", "Close");
 #ifndef QT_NO_NETWORKPROXY
     }
 #endif
